data/sql/updates/pending_db_world/rev_1631992050760579041.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631992050760579041');
+
+UPDATE 'quest_template_addon' SET 'ExclusiveGroup' = '7064' WHERE ('ID' IN (7064,7065));
```

Comment:
```suggestion
UPDATE `quest_template_addon` SET `ExclusiveGroup` = 7064 WHERE (`ID` IN (7064,7065));
```
Use backticks \` \` and dont use single quotes around non string values